addons/mendeley/models.py

```diff
@@ -0,0 +1,270 @@
+# -*- coding: utf-8 -*-
+
+import time
+
+from django.db import models
+import mendeley
+from mendeley.exception import MendeleyApiException
+from oauthlib.oauth2 import InvalidGrantError
+
+from addons.base.models import BaseOAuthUserSettings, BaseCitationsNodeSettings
+from website.addons.mendeley.serializer import MendeleySerializer
+from website.addons.mendeley import settings
```

Comment:
I've been trying to move the settings from their settings files into the app config. It can probably wait since we can't totally get rid of the old add-on anyway yet. Maybe add a todo?